crates/bevy_pbr/src/light.rs

```diff
@@ -727,7 +729,10 @@ pub(crate) fn assign_lights_to_clusters(
             ),
     );
 
-    if lights.len() > MAX_POINT_LIGHTS {
+    // NOTE: Clustered-forward rendering requires 3 buffer bindings so only use storage buffers
+    // if at least 3 are supported
+    let use_storage_buffers = render_device.limits().max_storage_buffers_per_shader_stage >= 3;
```

Comment:
This logic being repeated in a completely different place generally makes me nervous; what happens when we need to change it?

Reply:
I added a bitflags struct and a constructor that checks the number of required storage buffers per stage. I made that number (3) a constant. It's better, and good enough in my opinion, but I suspect it will change over time as I think the bitflags struct could encompass "Should we use uniform buffers, or storage buffers, and/or bindless for textures, or...)" but I'm not sure at the moment. So I'm solving the problem at hand in _a_ way, and when we understand more of what information is needed as features are built out, I expect this would be refactored.

I don't like doing that but, in my opinion, we can't really hold up implementation of things until everything is understood when understanding what the usage requirements are will probably remain unknown for some months yet.